.murdock

```diff
@@ -2,16 +2,19 @@
 
 : ${TEST_BOARDS_AVAILABLE:="samr21-xpro"}
 : ${TEST_BOARDS_LLVM_COMPILE:="iotlab-m3 native nrf52dk mulle nucleo-f401re samr21-xpro slstk3402a"}
+: ${TEST_BOARDS_LLVM_COMPILE:="native"}
```

Comment:
I think this is a merge error... Seems redundant (and unrelated) to re-set that (already set) variable.